tests/test_mapper.py

```diff
@@ -43,9 +43,6 @@ def setUp(self):
         ql.set_option('clifford_premapper', 'yes')
         ql.set_option('clifford_postmapper', 'yes')
         ql.set_option('mapper', 'minextendrc')
```

Comment:
What about tests in test_mapper that test initial placement?

Reply:
Those tests don't compare the compiler output to a reference.

I do note that initial placement was previously disabled because of the license issues, and thus was not running in CI.
Those tests did not make use of initial placement before, they just output the following log line:
`InitialPlace support disabled during OpenQL build [DONE]` and IP was no-op.

That's the big issue with on/off preprocessor variables, they likely result in uncovered code in CI, except if the CI setup follows.

I can adapt test_string in mapper.cc and put the same thing in the new unit test. I've already added a similar tests that requires a log of optimization to find a perfect mapping so we can discuss relevance of this test later today.